stubs/pyinstaller/PyInstaller/building/build_main.pyi

```diff
@@ -0,0 +1,28 @@
+# Referenced in: https://pyinstaller.org/en/stable/hooks.html?highlight=get_hook_config#PyInstaller.utils.hooks.get_hook_config
+# Not to be imported during runtime, but is the type reference for hooks and analysis configuration
+
+from _typeshed import StrOrBytesPath
+from collections.abc import Iterable
+
+from PyInstaller.building.datastruct import Target  # type: ignore[import]
```

Comment:
Why are all the `pyinstaller` imports `type: ignore`d in this PR?

Reply:
I was getting this error for PyInstaller.building and PyInstaller.depend
```
stubs\pyinstaller\PyInstaller\depend\imphookapi.pyi:6: error: Skipping analyzing "PyInstaller.building.build_main": module is installed, but missing library stubs or py.typed marker
stubs\pyinstaller\PyInstaller\building\build_main.pyi:7: error: Skipping analyzing "PyInstaller.building.datastruct": module is installed, but missing library stubs or py.typed marker
stubs\pyinstaller\PyInstaller\building\build_main.pyi:7: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
stubs\pyinstaller\PyInstaller\utils\hooks\__init__.pyi:11: error: Skipping analyzing "PyInstaller.depend.imphookapi": module is installed, but missing library stubs or py.typed marker
```

I think I was just missing some `__init__.pyi` files.

Reply:
Oh, that's a classic! Yeah, mypy struggles with namespace packages unless you have the `--namespace-packages` flag specified (which should be the default, see https://github.com/python/mypy/pull/9636).